### ISSUE TITLE
Align correlation images from different sessions

### ISSUE THREAD

**Issue by akatav** [flatironinstitute/CaImAn#764]
We have extracted active components from each session with only=True as follows:
        `active_Cells_0=extract_active_components(assign_forw,[0], only=True)`
       `active_Cells_1=extract_active_components(assign_forw,[1], only=True)` and so on.
We would like to plot the active components thus obtained from each session in the same plot. However, the correlation images for each session appear misaligned with each other. So, plotting each sessions' active components with CI set to some session's correlation image is not that effective. 
Is it a good idea to first align the correlation images from each session using some image alignment algorithm ? Does caiman provide any support for this?
Thanks

**Comment by epnev** [comment]
@akatav If you look at the [documentation](https://caiman.readthedocs.io/en/master/core_functions.html#rois) you'll see that this is already happening by default through the `align_flag` flag. The components get with respect to the FOV of the last session.

**Comment by akatav** [comment]
@epnev sure, i did set the align_flag to True but it does not seem to help, unfortunately. 
There are some blood vessels that are imaged  as well and using them as landmarks, we can clearly see that the correlation images from the different sessions obtained after the calls to register_ROIs() (we experimented with/without setting "use_opt_flow") are not aligned well to each other. We also set the shifts_opencv=False as setting it to True gives an empty xy_grid value causing error. 

For instance, these are the correlation images from subsequent sessions returned from register_ROIs: 
![corr](https://user-images.githubusercontent.com/721570/84242229-8326ce00-ab3b-11ea-8b45-5edfa400f285.png)
Clearly, sessions 2 and 3 are not aligned, for instance. 

In our experiments, the FOV obtained from the microscope (mounted on the mouse head) seems to have shifted a little bit between sessions. It will be great if there's something in caiman for handle these shifts during multisession registration.
